### PR TITLE
Add more date patterns

### PR DESCRIPTION
**Pull request by FriedEgg**

### CODE REVIEW

File: src/main/java/com/epam/parso/impl/SasFileConstants.java [hunk 1]
```diff
@@ -937,12 +937,48 @@
      * The date formats to store the day, month, and year. Appear in the data of the
      * {@link SasFileParser.FormatAndLabelSubheader} subheader and are stored in {@link com.epam.parso.Column#format}.
      */
-    List<String> DATE_FORMAT_STRINGS = Arrays.asList("YYMMDD", "MMDDYY", "DDMMYY", "DATE", "E8601DA");
+    List<String> DATE_FORMAT_STRINGS = Arrays.asList(
```
Comment (aborg0) [inline src/main/java/com/epam/parso/impl/SasFileConstants.java]: Shouldn't this be a `Set`? (Also `DATE_TIME_FORMAT_STRINGS`)

**Comment by Yana-Guseva** [comment]
We need to support Java 1.6 and this version doesn't support ISO 8601 time zone. 
So we can't use patterns which contain 'X' because it's avaliable since Java 7.

File: src/main/java/com/epam/parso/impl/SasFileConstants.java (hunk 1 above)
Comment (FriedEgg) [inline src/main/java/com/epam/parso/impl/SasFileConstants.java]: Changed it to a HashSet 85c438d8b99d54d175bc44392bd6b64bcf94e47c

**Comment by PetrTsurinov** [comment]
Thanks!

**Pull request merged**